npc/namespace.go

```diff
@@ -471,7 +471,17 @@ func (ns *ns) deleteNamespace(obj *coreapi.Namespace) error {
 	}
 
 	// Remove namespace ipset from any matching namespace selectors
-	return ns.nsSelectors.delFromMatching(obj.ObjectMeta.UID, obj.ObjectMeta.Labels, string(ns.allPods.ipsetName))
+	err := ns.nsSelectors.delFromMatching(obj.ObjectMeta.UID, obj.ObjectMeta.Labels, string(ns.allPods.ipsetName))
+	if err != nil {
+		return err
+	}
+
+	if !ns.legacy {
+		// Delete defaultAllowIPSet
+		return ns.ips.Destroy(ns.defaultAllowIPSet)
```

Comment:
nitpick: that comment is rather redundant.

Reply:
removed